.github/CODEOWNERS

```diff
@@ -339,6 +339,7 @@ yarn.lock                                                @getsentry/owners-js-de
 
 /src/sentry/scim/                                     @getsentry/enterprise
 /tests/sentry/api/test_scim*.py                       @getsentry/enterprise
+/src/sentry/tasks/integrations/github/pr_comment.py   @getsentry/enterprise @AniketDas-Tekky
```

Comment:
Why did you put yourself on it?

Reply:
It's still in development, I want to be personally notified about issues that come up in Sentry.

Reply:
I'll leave it as enterprise only after a few weeks of GA